packages/util/test/bytes.spec.ts

```diff
@@ -140,7 +140,7 @@ describe('bytesToHex', () => {
   it('empty Uint8Array', () => {
     const bytes = new Uint8Array()
     const hex = bytesToHex(bytes)
-    assert.strictEqual(hex, '0x')
+    assert.equal(hex, '0x')
```

Comment:
Why `strictEqual` -> `equal`?

Reply:
For a string like '0x' it seemed irrelevant, arguably now that I'm reading this: https://vitest.dev/api/assert.html#equal I'm thinking perhaps we should just move to strictEqual everywhere? I had just quickly glanced and assumed it meant "deepEqual" (which is for objects, so changedit to equal). 

We have banned non-strict equality in the monorepo I think so it would be natural for our testing suites to adjust for that. wdyt?

Reply:
Yes so the suggestion is to move to `strictEqual` everywhere, right (scope: other PR)? The change here is from `strictEqual` to `equal` (so less strict)

Reply:
Indeed. Created #4002 for it. It actually doesn,t matter too much because TS automatically types the method strictly (even if the method itself does not expect it), but nice to have nevertheless.